.changeset/lucky-pillows-stare.md

```diff
@@ -0,0 +1,83 @@
+---
+"@salt-ds/lab": patch
+---
+
+Updated the `Calendar` API to use a `multiselect` prop instead of `selectionVariant="multiselect"`
+
+Previously, the `Calendar` component used `selectionVariant="multiselect"` to enable the selection and de-selection of multiple single dates, with the de-selection logic embedded within the component.
+
+To extend multiple selection capabilities to all selection variants, we have revised the API to include a `multiselect` boolean prop. This change removes the need for `selectionVariant="multiselect"`.
+
+Now, you can apply the multiselect property in `Calendar` alongside `selectionVariant="single`", `selectionVariant="range"`, or `selectionVariant="offset"`.
+
+This update allows us to offer a consistent multiple selection API across all date selection variants.
+
+```diff
+<Calendar
+-  selectionVariant="multiselect"
++  selectionVariant="single"
++  multiselect
+>
+  <CalendarNavigation />
+  <CalendarGrid />
+</Calendar>
+```
+
+To de-select a previous selection, control is provided through a new `select` prop that enables control over the current selection based on the day that has been selected by the user.
+
+```diff
+function selectMultiselectSingle<TDate extends DateFrameworkType>(
+  dateAdapter: SaltDateAdapter<TDate>,
+  previousSelectedDate: SingleDateSelection<TDate>[],
+  newDate: TDate,
+) {
+  const newSelection = previousSelectedDate.filter(
+    (previousSingleDate) =>
+      !dateAdapter.isSame(previousSingleDate, newDate, "day"),
+  );
+  if (previousSelectedDate.length !== newSelection.length) {
+    return newSelection;
+  }
+  return [...previousSelectedDate, newDate];
+}
+
+<Calendar
+-  selectionVariant="multiselect"
++  selectionVariant="single"
++  multiselect
++   select={(
```

Comment:
I'm still unsure about the naming, wondering if we should call it something like selectionModifier or getSelection